consensus/types/src/chain_spec.rs

```diff
@@ -689,9 +689,10 @@ fn default_bellatrix_fork_epoch() -> Option<MaybeQuoted<Epoch>> {
 }
 
 fn default_terminal_total_difficulty() -> Uint256 {
-    "115792089237316195423570985008687907853269984665640564039457584007913129638912"
-        .parse()
-        .unwrap()
+    Uint256::from_dec_str(
+        "115792089237316195423570985008687907853269984665640564039457584007913129638912",
+    )
+    .unwrap()
```

Comment:
you could have this as a constant (`const fn default_terminal_total_difficulty() -> Uint256`) and in the test verify via this parsing that the constant is correct
```suggestion
    Uint256([
        18446744073709550592,
        18446744073709551615,
        18446744073709551615,
        18446744073709551615,
    ]);
```